aten/src/ATen/native/ReduceOps.cpp

```diff
@@ -232,6 +232,26 @@ Tensor& cumprod_out(Tensor& result, const Tensor& self, int64_t dim, c10::option
   return result;
 }
 
+void cummax_helper_cpu(const Tensor& self, Tensor& values, Tensor& indices, int64_t dim) {
```

Comment:
consider unifying cummax and cummin helper and templating them on the comparison function

Reply:
yeah that's a good point. however can a function in native_functions.yaml be templated?

Reply:
No they cannot be, but you can call non-templated 1-line wrapper from native_functions that will call into actual templated implementation.